### ISSUE TITLE
[A11yInspect] Fail: 1.3.1 A in /home/runner/work/test-app/test-app/src/sample.jsx

### ISSUE THREAD

**Issue by sitaram-r** [sitaram-r/test-app#12]
## ❌ Accessibility Fail

**Rule:** 1.3.1 A
**File:** `/home/runner/work/test-app/test-app/src/sample.jsx`
**Line:** 1, **Column:** 1

### Description
Missing table header (Major)

### Context
```html
<table>
          <tbody><tr>
            <td>Name</td>
            <td>Age</td>
          </tr>
        </tbody></table>
```

### Selector
`html > body > div > div > video > audio > table:nth-of-type(1)`

---
*Generated by [A11yInspect](https://github.com/barrierbreak/vscode-scally)*
